features/top-level-await.yml

```diff
@@ -2,11 +2,16 @@ name: Top-level await
 description: The `await` keyword, when used at the top level of a module (outside of an async function), delays parent module execution until after a promise fulfills.
 spec: https://tc39.es/ecma262/multipage/ecmascript-language-functions-and-classes.html#sec-async-function-definitions
 group: javascript
-# TODO: https://github.com/web-platform-dx/web-features/issues/1971
-# Status changed: https://github.com/web-platform-dx/web-features/pull/2921/
-# 2025-05-02 — high → false — In Safari, simultaneous imports of a module with a top-level await fails with a ReferenceError.
-# References:
-# - https://github.com/mdn/browser-compat-data/pull/26510
-# - https://bugs.webkit.org/show_bug.cgi?id=242740
-compat_features:
-  - javascript.operators.await.top_level
+# TODO: This feature may have regressed its status. See: https://github.com/web-platform-dx/web-features/issues/2957
+status:
+  baseline: high
+  baseline_low_date: 2021-09-20
+  baseline_high_date: 2024-03-20
+  support:
+    chrome: "89"
+    chrome_android: "89"
+    edge: "89"
+    firefox: "89"
+    firefox_android: "89"
+    safari: "15"
+    safari_ios: "15"
```

Comment:
This is an editorial override to unwind (at least temporarily) the change made at https://github.com/web-platform-dx/web-features/pull/2921/files/c5f568809957d243ef1558568c7d1f01c9b87c7a#r2071895308

See https://github.com/web-platform-dx/web-features/issues/2957